dnsping.py

```diff
@@ -197,20 +201,12 @@ def main():
 
     dnsserver = validate_server_address(dnsserver, af)
 
-    if use_edns:
-        query = dns.message.make_query(qname, rdatatype, dns.rdataclass.IN,
-                                       use_edns=True, want_dnssec=want_dnssec,
-                                       ednsflags=dns.flags.edns_from_text('DO'), payload=8192)
-    else:
-        query = dns.message.make_query(qname, rdatatype, dns.rdataclass.IN,
-                                       use_edns=False, want_dnssec=want_dnssec)
-
     response_time = []
     i = 0
 
-    print("%s DNS: %s:%d, hostname: %s, proto: %s, rdatatype: %s, flags: %s" %
-          (__progname__, dnsserver, dst_port, qname, proto_to_text(proto), rdatatype,
-           dns.flags.to_text(query.flags)), flush=True)
+    print("%s DNS: %s:%d, hostname: %s, force_miss: %s, proto: %s, rdatatype: %s, flags: %s" %
+          (__progname__, dnsserver, dst_port, qname, force_miss, proto_to_text(proto), rdatatype,
+           dns.flags.to_text(dns.flags.RD)), flush=True)
```

Comment:
I'm not sure how the flags here should be done. Initially I kept the code between line 200 and 207 to create a "dummy request" that I could then use for `dns.flags.to_text(dummyquery.flags)` but that didn't feel very nice. So I tried to see which of the options would actually change the flags from `RD`, but couldn't see or think of any case that this would happen? 

Tried to look at what `make_query` does https://github.com/rthalley/dnspython/blob/v2.1.0/dns/message.py#L1433 

and also tried the options for `dnsping` to see if any of them changed the outgoing flags, but couldn't see it. If there is an option that changes this, please let me know.